src/commands/approve/utils/getRawThreadName.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { Channel, ChannelType } from 'discord.js'
+import { getRawThreadName } from './getRawThreadName'
+
+describe('getRawThreadName', () => {
+  describe('when the message match the regex', () => {
+    describe('and the channel is a public thread', () => {
```

Comment:
eita, precisa dessa cascata de describe msm? agrupa tudo em um 
```suggestion
describe('getRawThreadName when the message match the regex and the channel is a public thread', () => {

```